test/e2e/externalservices_kubernetes_test.go

```diff
@@ -192,23 +198,33 @@ metadata:
 		err := YamlK8s(fmt.Sprintf(trafficRoute, es2))(cluster)
 		Expect(err).ToNot(HaveOccurred())
 
-		pods, err := k8s.ListPodsE(
-			cluster.GetTesting(),
-			cluster.GetKubectlOptions(TestNamespace),
-			metav1.ListOptions{
-				LabelSelector: fmt.Sprintf("app=%s", "demo-client"),
-			},
-		)
-		Expect(err).ToNot(HaveOccurred())
-		Expect(pods).To(HaveLen(1))
-
-		clientPod := pods[0]
-
 		stdout, stderr, err := cluster.ExecWithRetries(TestNamespace, clientPod.GetName(), "demo-client",
 			"curl", "-v", "-m", "3", "--fail", "http://external-service.mesh")
 		Expect(err).ToNot(HaveOccurred())
 		Expect(stderr).To(ContainSubstring("HTTP/1.1 200 OK"))
 		Expect(stdout).To(ContainSubstring("HTTPS"))
 	})
 
+	It("Should disable passthrough", func() {
+
+		_, err := retry.DoWithRetryE(cluster.GetTesting(), "passthrough access to service", 5, DefaultTimeout, func() (string, error) {
+			_, _, err := cluster.Exec("", "", "demo-client",
+				"curl", "-v", "-m", "3", "--fail", "http://externalservice-http-server.externalservice-namespace")
+			if err != nil {
+				return "", err
+			}
+
+			return "", nil
+		})
+		Expect(err).To(HaveOccurred())
+
+		err = YamlK8s(fmt.Sprintf(meshDefaulMtlsOn, "true"))(cluster)
```

Comment:
Just a note for whoever will take care of E2E tests eventually. We need proper builders for our entities. Something like this:
```
MeshBuilder().WithPassThrough(false).WithMTLS().KubeYAML()
```
Copy-pasting those YAMLs with parameters over many test is not the best idea IMHO.